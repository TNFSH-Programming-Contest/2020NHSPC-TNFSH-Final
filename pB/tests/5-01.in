4
-471.274 515.728
-616.789 515.728
-616.789 -492.135
-471.274 -492.135
-578.134 -207.581
