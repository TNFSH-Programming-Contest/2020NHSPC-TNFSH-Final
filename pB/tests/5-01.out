1679479.628
